api_spec.yaml

```diff
@@ -3246,9 +3246,496 @@ paths:
           $ref: '#/components/responses/401'
         '500':
           $ref: '#/components/responses/500'
-          
-      
+  
+  /recipe/multitenancy/tenant:
+    put:
+      tags:
+        - Multitenancy Recipe
+      description: |
+        Creates or updates a tenant.
+
+        If `tenantId` is not specified, `defaultTenantId` will be used.
+
+        If `tenantId` is specified and is not equal to `defaultTenantId`, SuperTokens subscription license key is required.
+
+        If creating a new tenant, only the login methods set to true will be enabled and rest will be disabled by default.
+
```

Comment:
API that returns tenant info needs to also return its resolved / normalised config